aiotg/bot.py

```diff
@@ -100,6 +101,34 @@ def run(self):
         except KeyboardInterrupt:
             self.stop()
 
+    def run_with_reloader(self, **kwargs):
```

Comment:
Sure, or maybe add a `reload` flag to the main `run()` function. What do you prefer?